src/components/dev-hub/feedback/feedback-container.js

```diff
@@ -11,21 +18,134 @@ const feedbackItems = graphql`
     }
 `;
 
-const FeedbackContainer = () => {
-    const data = useStaticQuery(feedbackItems);
+const StyledButtonContainer = styled('div')`
+    display: flex;
+    justify-content: center;
+    margin-bottom: 4px;
+`;
+
+const StyledForm = styled('form')`
```

Comment:
![Screen Shot 2021-04-14 at 11 22 38 AM](https://user-images.githubusercontent.com/9064401/114736049-c3cf1500-9d13-11eb-8e0f-cf9902b4572a.png)

Perhaps we should use the `narrow` prop on our input or revert any additional styling changes as the label should not cut off the element below. This works well on other forms so we should be able to omit some of the changes here.